programs/sbf/tests/programs.rs

```diff
@@ -107,15 +121,7 @@ fn process_transaction_and_record_inner(
             None,
         )
         .0;
-    let CommittedTransaction {
-        inner_instructions,
-        log_messages,
-        status,
-        ..
-    } = commit_results.swap_remove(0).unwrap();
-    let inner_instructions = inner_instructions.expect("cpi recording should be enabled");
-    let log_messages = log_messages.expect("log recording should be enabled");
-    (status, inner_instructions, log_messages)
+    commit_results.swap_remove(0)
```

Comment:
minor:
We know the length is 1. `pop` & `unwrap` would probably be slightly better here since then we don't need to do the copy.